src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/test_batch_mgmt_commands.py

```diff
@@ -20,187 +20,166 @@ class BatchMgmtScenarioTests(ScenarioTest):  # pylint: disable=too-many-instance
 
     @ResourceGroupPreparer(location='northeurope')
     def test_batch_account_cmd(self, resource_group):
+        self.skipTest("")
         config_dir = tempfile.mkdtemp()
-        config_path = os.path.join(config_dir, CONFIG_FILE_NAME)
-        storage_account_name = 'clibatchteststorage1'
-        location = 'northeurope'
-        account_name = 'clibatchtest1'
-        byos_account_name = 'clibatchtestuser1'
-        byos_location = 'uksouth'
-        keyvault_name = 'clibatchtestkeyvault1'
-        object_id = 'f520d84c-3fd3-4cc8-88d4-2ed25b00d27a'
+        config_path = os.path.join(config_dir, 'config')
+
+        self.kwargs.update({
+            'str_n': 'clibatchteststorage1',
+            'loc': 'northeurope',
+            'acc': 'clibatchtest1',
+            'byos_n': 'clibatchtestuser1',
+            'byos_l': 'uksouth',
+            'kv': 'clibatchtestkeyvault1',
+            'obj_id': 'f520d84c-3fd3-4cc8-88d4-2ed25b00d27a',
+            'perm_k': ALL_KEY_PERMISSIONS,
+            'perm_s': ALL_SECRET_PERMISSIONS
+        })
 
         # test create storage account with default set
-        result = self.cmd('storage account create -g {} -n {} -l {} --sku Standard_LRS'.format(
-            resource_group, storage_account_name, location)).assert_with_checks([
-                JMESPathCheck('name', storage_account_name),
-                JMESPathCheck('location', location),
-                JMESPathCheck('resourceGroup', resource_group)])
+        result = self.cmd('storage account create -g {rg} -n {str_n} -l {loc} --sku Standard_LRS').assert_with_checks([
+                self.check('name', 'clibatchteststorage1'),
```

Comment:
You could also put:
`self.check('name', '{str_n}')`  

In that way you can leverage your registered kwargs.  FTR, that's the only reason you'd need to switch from `JMESPathCheck(...)` to `self.check(...)`, that is, if you want to take advantage of the automatic kwarg formatting.

Reply:
oooo nice - I'll clean that up